VideoLocker/res/values/styles.xml

```diff
@@ -299,13 +298,13 @@
 
     <!-- Spinner with the popup style -->
     <style name="edX.Widget.ListPopupWindow" android:parent="@style/Widget.AppCompat.ListPopupWindow">
-        <item name="android:popupBackground">?android:attr/popupBackground</item>
+        <item name="android:popupBackground">@drawable/popup_menu</item>
```

Comment:
the drawable is only being used as a `background`, why not name it something like `popup_menu_bg`